docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -1,3 +1,27 @@
+= Deprecation of 'legacy' cookie behavior
+
+As part of the https://privacysandbox.com/[Privacy Sandbox] effort, Keycloak is making changes to the way cookies are handled. Specifically we are preparing for the https://developers.google.com/privacy-sandbox/blog/cookie-countdown-2023oct[end of third-party cookies], and are looking to improve the consistency, privacy and security of how cookies are handled by default within Keycloak. This has several implications for how Keycloak is configured and used:
+
+== Removal of cookies suffixed in `_LEGACY`
+
+Since the introduction of the https://web.dev/articles/samesite-cookies-explained[`SameSite`] attribute on cookies provided by Keycloak, another set of 'legacy' cookies (name suffixed in `_LEGACY`) were also introduced in order to support older browsers that were https://www.chromium.org/updates/same-site/incompatible-clients/[not compatible] with this attribute. Since the `SameSite` attribute is now https://caniuse.com/same-site-cookie-attribute[widely supported], this behavior is now deprecated, so that only the cookies using the `SameSite` attribute remain.
+
+== `SameSite` attribute set to `Strict` by default
```

Comment:
Is this title accurate or it is more about our future plans? We don't necessarily need to force `strict` but also consider making the attribute configurable, as per discussions we had in the past.

Reply:
The documentation updates here reflect what I intend the implementation to be under this PR (yet to be implemented). 

> We don't necessarily need to force `strict` but also consider making the attribute configurable, as per discussions we had in the past.

It would still be 'overwritable' in code for specific use-cases, but not globally configurable if that is what you mean here. Is there a link to these previous discussions somewhere that can be referenced?

Future changes in Chrome will move all these to `Strict` by default, as part of the 3rd-party cookie phase-out. So this is just about getting ahead of these changes by forcing it as the default.

Reply:
> Future changes in Chrome will move all these to Strict by default, as part of the 3rd-party cookie phase-out. So this is just about getting ahead of these changes by forcing it as the default.

Yeah, but only if we don't set the attribute, right? My point is that I'm not sure if we should do the `Strict` now because, AFAIK, it is perfectly fine to go with `secure` and `none`. And TBH, I think that should be the default as a balance between security and usability.

I'll need to go over the closed/open issues to filter discussions ...

Reply:
> Yeah, but only if we don't set the attribute, right?

To my knowledge also if we don't set the attribute, it will eventually not be possible to share cookies in a third-party context.

Reply:
Not really for applications running thin the same domain, right?

We have open issues about making it possible to configure our cookies and use `strict`. As I mentioned before, for now we should keep using `none` and have a follow-up to properly discuss the best/secure default settings. But I think it makes sense to make this configurable too.